app/src/main/java/com/github/h3lp3rs/h3lp/view/helprequest/helpee/HelpeeSelectionActivity.kt

```diff
@@ -116,7 +107,7 @@ class HelpeeSelectionActivity : AppCompatActivity() {
                 .setOnClickListener {
                     // In case the getCurrentLocation failed (for example if the location services aren't
                     // activated, currentLocation is still null and the returned phone number will be the
-                    // default emergency phone number
+                    // default emergency phone number)
```

Comment:
```suggestion
                    // default emergency phone number
```

Reply:
The ')' is correct here it is open in the first line of the comment